packages/react/src/ReactSharedInternals.js

```diff
@@ -25,4 +27,8 @@ if (__DEV__) {
   ReactSharedInternals.ReactCurrentActQueue = ReactCurrentActQueue;
 }
 
+if (enableServerContext) {
+  ReactSharedInternals.globalServerContextRegistry = globalServerContextRegistry;
```

Comment:
Tiny nit. Just because this can't be minified. Maybe a shorter name? "ContextRegistry"? "global" is kind of implied, since all things in here are global. The "Server" part is just one kind of context that happens to have been registered.